warp10/src/main/java/io/warp10/script/mapper/MapperAbs.java

```diff
@@ -39,6 +39,10 @@ public Object apply(Object[] args) throws WarpScriptException {
     long[] elevations = (long[]) args[5];
     Object[] values = (Object[]) args[6];
 
+    if (0 == values.length) {
+      return new Object[] {0L, GeoTimeSerie.NO_LOCATION, GeoTimeSerie.NO_ELEVATION, null};
```

Comment:
This case was already handled correctly by this mapper. Also this change may make the MAP delete a value at epoch 0.

Reply:
It is consistent with all other existing mapper implementation. Look at MapperAdd.
(I mark as resolved the related conversations)

![image](https://user-images.githubusercontent.com/37228740/195601055-45d9de4b-a7bb-44b8-b2fd-3c02b7f7d0f2.png)